### PR TITLE
Added a self hosting option for AWS

### PR DESCRIPTION
**Pull request by AlaShibanAtKlo**
## Summary of change
We're fans of Supertokens and added support to install it on AWS using stacksnap 

## Related issues
- N/A

## Checklist
- [X] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [X] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [X] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

### CODE REVIEW

File: v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (rishabhpoddar) [inline v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx:1]: Please modify the [sidebar.json](https://github.com/supertokens/docs/blob/a4e2261fb53b295834f55239747e4a40835d18c6/v2/passwordless/sidebars.js) to add this page

File: v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---  
+id: aws-setup-with-stacksnap  
+title: AWS Setup with Stacksnap  
+hide_title: true  
+---
+
```
Comment (rishabhpoddar) [inline v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx:6]: please replicate this across the following recipes:
- emailpassword
- thirdparty
- thirdparty + emailpassword
- thirdparty + passwordless
- session

File: v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---  
+id: aws-setup-with-stacksnap  
+title: AWS Setup with Stacksnap  
+hide_title: true  
+---
```
Comment (rishabhpoddar) [inline v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx:5]: neds to be replicated in the custom ui section of the recipe as well please.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Comment by rishabhpoddar** [comment]
Hey @AlaShibanAtKlo ! This is seriously cool! Thank you. I have added a few comments.

**Comment by AlaShibanAtKlo** [comment]
> Hey @AlaShibanAtKlo ! This is seriously cool! Thank you. I have added a few comments.

I made the changes, I believe i caught all the placements 🤞thanks @rishabhpoddar !

**Comment by AlaShibanAtKlo** [comment]
@rishabhpoddar ok I fixed everything and I think everything is passing now 🤞

**Comment by rishabhpoddar** [comment]
Thanks @AlaShibanAtKlo ! Please feel free to add yourself to the [supertokens-core readme](https://github.com/supertokens/supertokens-core) :)

**Pull request merged**